requirements/dev.txt

```diff
@@ -3,5 +3,4 @@
 pip
 wheel
 flake8
-sphinx
-flask-script
```

Comment:
please use also 
`pipenv uninstall flask-script`
to uninstall the packages properly from the pipfile as well

Reply:
done